HIP/hip-583.md

```diff
@@ -2,221 +2,266 @@
+   calculated on the consensus nodes using the `ECDSA` key provided in the auto create `CryptoTransfer` transaction.
+
+Examples of each format are noted below:
+(Please NOTE: the `alias` is a binary format when encoded on-wire for protobuf. We represent the bytes as base-32
+encoded ASCII for clarity in the documentation, but in reality these are raw bytes)
+
+| Identifier Type                | Example                                                      |
+|--------------------------------|--------------------------------------------------------------|
+| Hedera Account ID              | 0.0.10                                                       |
+| Hedera Account Long-Zero Alias | 0.0.000000000000000000000000000000000000000a                 |
+| Hedera Key Alias               | 0.0.CIQNOWUYAGBLCCVX2VF75U6JMQDTUDXBOLZ5VJRDEWXQEGTI64DVCGQ  |
+| EVM Address Alias              | 0.0.b794f5ea0ba39494ce839613fffba74279579268                 |
+
+TODO: Are all three of these base-32 encoded? Should use consistent encoding
+
+To provide greater context for this HIP some additional terms useful:
+
+- `Ethereum-native wallets` - wallets that utilize the Ethereum JSON-RPC APIs implemented by the JSON-RPC Relay.
+  Accounts created via these wallets are `ECDSA` derived keys and use the `evm address alias` format.
+- `Hedera-native wallets` - wallets that utilize the Hedera SDK or gRPC HAPI calls to interact with Hedera consensus
+  and mirror nodes. Historically, accounts created via these wallets are `ED25519` based and usually identify the
+  account via the `Hedera Account Id`.
+- `Hollow account` - a partially created account which lacks a public key but does have a valid `shard.realm.num`
+  accountId _and_ is auto-created with an `evm address alias`. Such an account may receive assets but may not take
+  part in transactions that require its signature, until it is promoted to being a "full" account.
+- `auto-create` - an account creation flow in which the `key alias` or `evm address alias` that a crypto-transfer
+  is addressed to is used to automatically create a new account (if none exists) without an explicit `CryptoCreate`
+  transaction being submitted. We call it `hollow-create` when the `evm address alias` is used, since it results
+  in the creation of a `Hollow account`.
+
+With an Ethereum-native wallet, you can transfer the native currency (in our case, HBAR), or tokens, to an `evm address`
+that does not yet exist. If nobody in the universe has the private key for that account, the value held by that account
+is effectively locked forever. If somebody does have the keys to that account, they can access the account and control
+it. Since Ethereum-native wallets do not understand the `0.0.10` style format of a Hedera account ID, it is necessary
+to support the creation of accounts with an `evm address alias` to enable compatibility between Ethereum-native
+wallets and Hedera-native wallets and all accounts.
 
 ## User stories
```

Comment:
I like it. I think from an implementation perspective it is covered by this use case I added today (before I saw your comment):

> 7. As a Hedera user with a `Hedera-native wallet`,  I would like to transfer HBAR or tokens to another account using
   only the recipient's `evm address alias`.

However I think it is good to get your user story as well since it may be useful for documentation and outreach and shows one reason why we are doing what we are doing.